src/test/java/com/snowflake/kafka/connector/ConnectorConfigValidatorTest.java

```diff
@@ -495,7 +495,6 @@ public void testStreamingEmptyFlushTime() {
         IngestionMethodConfig.SNOWPIPE_STREAMING.toString());
     config.put(Utils.SF_ROLE, "ACCOUNTADMIN");
     config.remove(SnowflakeSinkConnectorConfig.BUFFER_FLUSH_TIME_SEC);
-    config.put(SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER, "false");
```

Comment:
we could keep that, it will be later easier to set the default back to true

Reply:
But leaving the line may suggest we put a meaningful configuration value in this test. Deleting this makes it clear we use the default one.